extremeflash/ws.py

```diff
@@ -112,7 +112,7 @@ def determine_openwrt_boot_params(model):
 def bootup_set_boot_openwrt(ser: serial.Serial, dryrun: bool = False) -> str:
     ser.write(b"printenv\n")
     time.sleep(1)
-    printenv_return = ser.read(ser.in_waiting).decode("ascii")
+    printenv_return = ser.read_all().decode("ascii")
```

Comment:
Is this change by accident?

Reply:
It is part of SerialBase:
https://github.com/pyserial/pyserial/blob/5fe2f9c568a91aa21aff129bc7cbf85fcfe8bad8/serial/serialutil.py#L662

There, you can also see that it does exactly the same as the replaced code :)